sdmetrics/single_table/privacy/dcr_utils.py

```diff
@@ -0,0 +1,162 @@
+"""Distance to closest record measurement functions."""
+
+import pandas as pd
+
+from sdmetrics.utils import is_datetime
+
+
+def _calculate_dcr_value(s_value, d_value, col_name, metadata, range=None):
```

Comment:
Can we rename `range` to something like `col_range` instead since `range` conflicts with the python function?